docs-website-src/content/_index.md

```diff
@@ -296,6 +296,70 @@ You can unlock the situation by using the `--clean` option.
    /scripts/elevate-cpanel --start
 ```
 
+### The CCS service will not start after elevate succeeds
+
+This can sometimes occur due to a failed schema update.  When this occurs, it
+is best to remove the CCS package(s), remove the home directory for the packages
+user, reinstall the package, and finally, ensure that the task queue completes
+before continuing:
```

Comment:
```suggestion
This can sometimes occur due to a failed schema update.  When this occurs, we recommend that you complete the following steps: 

1. Remove the CCS package(s), 
2. Remove the home directory for the packages user, 
3. Reinstall the package, 
4. Finally, ensure that the task queue completes before continuing

Use the following scripts to complete these steps:
```

Reply:
done